environment.yml

```diff
@@ -1,138 +1,11 @@
 name: simple
 channels:
-  - http://ssb.stsci.edu/astroconda
   - defaults
+  - conda-forge
+  - http://ssb.stsci.edu/astroconda
```

Comment:
Do we actually need conda-forge and astroconda? I thought the later was actually deprecated.